src/views/RecIM/components/List/Listing/index.jsx

```diff
@@ -412,6 +455,32 @@ const ParticipantListing = ({
             )}
           </Menu>
         )}
+        {isAdminPage && participant.IsCustom && (
+          <Menu
+            open={moreOptionsCustomOpen}
+            onClose={handleClickOff}
+            anchorEl={anchorCustomEl}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'right',
+            }}
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'right',
+            }}
+          >
+            <MenuItem
+              dense
+              onClick={() => {
+                editDetails(participant);
```

Comment:
```suggestion
                editDetails(participant);
```
editDetails of what? descriptive?